hover/effects/_keyframes.sass

```diff
@@ -0,0 +1,10 @@
+// Keyframes mixin to add browser prefixes to animations
+@mixin keyframes($animation-name)
+    @-webkit-keyframes #{$animation-name}
+        @content
+
+    @-moz-keyframes #{$animation-name}
```

Comment:
yeah, I wasn't sure about this one, I took out `ms` and `o` but thought I'd leave this just in case? It's probably not necessary...